pkg/build/buildkit/errors.go

```diff
@@ -94,7 +96,16 @@ func GetErrorMessage(err error, tag string) error {
 }
 
 func extractImageTagFromPullAccessDeniedError(err error) string {
-	re := regexp.MustCompile(`([a-zA-Z0-9\.\/_-]+): pull access denied`)
+	re := regexp.MustCompile(`([a-zA-Z0-9\.\/_-]+(:[a-zA-Z0-9]+)?): pull access denied`)
+	matches := re.FindStringSubmatch(err.Error())
+	if len(matches) > 1 {
+		return matches[1]
+	}
+	return ""
+}
+
+func extractImageTagFromNotFoundError(err error) string {
+	re := regexp.MustCompile(`([a-zA-Z0-9\.\/_-]+(:[a-zA-Z0-9]+)?): not found`)
```

Comment:
I think it is not getting properly the image name.

With the following `image` field in an Okteto manifest `ifbyol/pipeline-runner:go-getting-started`, it is printing the following error message

<img width="1422" alt="Screenshot 2024-11-20 at 09 39 06" src="https://github.com/user-attachments/assets/99ae8652-169f-4f22-a052-377b77d9d931">

So it seems it is only getting the tag of the image?

Reply:
yup, the expression for capturing the tag was not correct, fixed and added tests.